src/zadacha_15.py

```diff
@@ -0,0 +1,22 @@
+BOARD_SIZE = 8
```

Comment:
here you have 2 empty lines between constant and code, earlier you put 0 lines. Styling should be consistent.